generate

```diff
@@ -158,7 +158,7 @@ def recurse(aspect, level=0):
                 output += "* `" + subsub_name + " <" + subsub_name + "/README.rst>`_\n"
         else:
             output += "This aspect does not have any sub aspects.\n\n"
-
+        output += "\n"
```

Comment:
I also tried python3 generate but, that too is showing some error
![screenshot from 2017-03-09 22-00-37](https://cloud.githubusercontent.com/assets/21004581/23796975/919c1818-0595-11e7-8714-0a8c189bbd30.png)